src/driver/close.rs

```diff
@@ -1,29 +1,34 @@
 use crate::driver::Op;
 
 use crate::driver::op::{self, Completable};
-use std::io;
-use std::os::unix::io::RawFd;
+use std::{io, os::unix::io::RawFd};
 
 pub(crate) struct Close {
     fd: RawFd,
 }
 
 impl Op<Close> {
+    /// Close a file descriptor
```

Comment:
Comments unrelated to the PR, but helped me understand the whole Close shutdown saga whilst implementing